utils/src/main/scala/za/co/absa/hermes/utils/SparkTestBase.scala

```diff
@@ -30,5 +30,6 @@ trait SparkTestBase { self =>
     .config("spark.driver.bindAddress", "127.0.0.1")
     .config("spark.driver.host", "127.0.0.1")
     .config("spark.sql.hive.convertMetastoreParquet", false)
+    .config("fs.defaultFS","file:/")
```

Comment:
Just curious. What does this do?

Reply:
This makes the test actually use local filesystem and not a running HDFS.